docs/proposals/multiple-dns-provider-support.md

```diff
@@ -0,0 +1,86 @@
+# Multiple DNS Provider Support
+
+Authors: Michael Nairn @mikenairn
+
+Epic: https://github.com/Kuadrant/multicluster-gateway-controller/issues/189
+
+Date: 25th May 2023
+
+
+## Job Stories
+
+- As a developer, I want to use MGC with a domain hosted in one of the major cloud DNS providers (Google Cloud DNS, Azure DNS or AWS Route53) 
+- As a developer, I want to use  multiple domains with a single instance of MGC, each hosted on different cloud providers
+
+## Goals
+
+- Add ManagedZone and DNSRecord support for [Google Cloud DNS](https://cloud.google.com/dns/docs/)
```

Comment:
Of the 3 providers that don't currently have implementations in mgc, what priority/order would you put on them?

Reply:
:man_shrugging:

Reply:
Yeah not sure we can answer that without use feedback. I personally would prob go for google but I have no real reasoning behind that other than it looks to be a simpler integration

Reply:
Yeah, id agree google will be the easier of the two (not counting CoreDNS) to integrate with and probably the one to do first.